src/game.h

```diff
@@ -312,6 +314,10 @@ class game
          * Returns a `nullptr` if the monster could not be created.
          */
         monster *summon_mon( const mtype_id& id, const tripoint &p );
+        /**
+         * Summons an NPC.
```

Comment:
A function named "summon_npc" summons an NPC? That documentation is a pointless wast of time to read. The function name tells exactly the same. Either remove it completely or make it into something that *adds* information.